frontend/src/pages/Courses.jsx

```
@@ -68,8 +70,15 @@ const Courses = () => {
 
       {courses.map((course) => (
         <div
-          onClick={(e) => navigate(`/courses/${course._id}`)}
           key={course._id}
+          role="button"
```

Biome (1.9.4)

[error] The elements with the following roles can be changed to the following elements:
<button>
<input type="button">


For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)